packages/block-library/src/table-of-contents/edit.js

```diff
@@ -137,8 +137,11 @@ export default function TableOfContentsEdit( {
 	return (
 		<>
 			<nav { ...blockProps }>
-				<ol inert="true">
```

Comment:
I'm not sure why no one caught this regression. This makes the content completely inaccessible to keyboard users. Please don't do this unless you have a really good reason to.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert

Reply:
`inert` was added in several places in https://github.com/WordPress/gutenberg/commit/d50e613d110ea7ccd5de3f647c8dcf3b72bdda74; we maybe should take a look at each usage.